src/Sarif.Multitool/RebaseUriCommand.cs

```diff
@@ -35,24 +35,34 @@ public int Run(RebaseUriOptions rebaseOptions)
                     baseUri = new Uri(baseUri.ToString() + "/");
                 }
 
-                var sarifFiles = GetSarifFiles(rebaseOptions);
+                var rebaseUriFiles = GetRebaseUriFiles(rebaseOptions);
+
+                bool outputFilesCanBeCreated = true;
+                foreach (RebaseUriFile rebaseUriFile in rebaseUriFiles)
+                {
+                    outputFilesCanBeCreated &=
+                        DriverUtilities.ReportWhetherOutputFileCanBeCreated(
```

Comment:
>ReportWhetherOutputFileCanBeCreated [](start = 40, length = 35)

I'm a little surprised we don't create an overload of ReportWhetherOutputFileCanBeCreated which accepts an IEnumerable<outputfilepath>. Is this a special case? i.e. it's unlikely we'll require the overload anywhere else.

Reply:
This construct (loop over output files) occurs both here (`rebaseuri` command) and in the `absoluteuri` command, and in general it will occur in any command whose options class derives from `MultipleFilesOptionsBase`. So yes, it is a good opportunity to DRY this out.

---
In reply to: [312282499](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r312282499) [](ancestors = 312282499)

Reply:
Done.

---
In reply to: [312283038](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r312283038) [](ancestors = 312283038,312282499)